tests/entrypoints/openai/test_tools.py

```diff
@@ -0,0 +1,659 @@
+import json
+from typing import Dict, List, Optional
+
+import openai
+import pytest
+from huggingface_hub import snapshot_download
+from openai.types.chat import (ChatCompletionMessageParam,
+                               ChatCompletionToolParam)
+from typing_extensions import TypedDict
+
+from ...utils import VLLM_PATH, RemoteOpenAIServer
+
+
+class ServerConfig(TypedDict):
+    model: str
+    arguments: List[str]
+
+
+class TestConfig(TypedDict):
+    client: openai.AsyncOpenAI
+    model: str
+
+
+ARGS: List[str] = [
+    "--dtype",
+    "half",  # TODO change to BF16
+    "--kv-cache-dtype",
+    "fp8",
```

Comment:
Do you need these? I think you can just remove

Reply:
I need them for me because I'm on a Nvidia v100 Tesla 32GB which is small and which doesn't support `bfloat16` (I have to manually specify `half` for everything), but I will remove them as soon as I write that last test and mark the pr as `ready`